### DISCUSSION TITLE
i can use create_pool to query data,but why can't i query the new data that i manually inserted

### DISCUSSION THREAD

**Discussion by Ray8716397** [aio-libs/aiomysql#735] [Q&A] [answered]
i can use create_pool to query data,but why can't i query the new data that i manually inserted?

**Comment by evgenybf** [comment, marked as answer]
@Ray8716397 Could you provide an example? I have a suspicion, that the new data you manually inserted is not committed (autocommit=False or you started a transaction and forgot to call `COMMIT`)

**Reply by Nothing4You** [reply]
@Ray8716397 is this still relevant for you?
